MekHQ/src/mekhq/MekHQ.java

```diff
@@ -374,6 +375,11 @@ public void joinGame(Scenario scenario, List<Unit> meks) {
     }
 
     public void startHost(Scenario scenario, boolean loadSavegame, List<Unit> meks) {
+        startHost(scenario, loadSavegame, meks, null);
+    }
+
+    public void startHost(Scenario scenario, boolean loadSavegame, List<Unit> meks, BehaviorSettings autoResolveBehaviorSettings)
```

Comment:
Can we get some JavaDocs on these public methods. Generally, we're trying to improve our JavaDoc coverage, so if you edit or add a public method it's best to JavaDoc it. Private methods only need to be JavaDoc'd if the purpose is unclear or if there is something interesting to say, but I tend to JavaDoc everything for the sake of completeness.

Reply:
Yeap, can do.